test/sanity/ignore.txt

```diff
@@ -5280,13 +5280,7 @@ lib/ansible/modules/network/vyos/vyos_linkagg.py validate-modules:E326
 lib/ansible/modules/network/vyos/vyos_linkagg.py validate-modules:E337
 lib/ansible/modules/network/vyos/vyos_linkagg.py validate-modules:E338
 lib/ansible/modules/network/vyos/vyos_linkagg.py validate-modules:E340
-lib/ansible/modules/network/vyos/vyos_lldp.py future-import-boilerplate
-lib/ansible/modules/network/vyos/vyos_lldp.py metaclass-boilerplate
```

Comment:
These entries seem to be removed.

Reply:
As discussed this causing Error "unnecessary imports includes" So had to remove this